main/src/vtr_navigation/src/navigator.cpp

```diff
@@ -85,7 +85,7 @@ Navigator::Navigator(const rclcpp::Node::SharedPtr& node) : node_(node) {
 
   /// pose graph
   auto new_graph = node_->declare_parameter<bool>("start_new_graph", false);
-  graph_ = tactic::Graph::MakeShared(data_dir + "/graph", !new_graph,
+  graph_ = tactic::Graph::MakeShared(data_dir + "/graph", new_graph,
```

Comment:
Was this meant to be changed? This seems odd?

Reply:
No, I just did that for testing - seemed like an easy way to always have it make a new graph by default. I've changed it back now.